javascript-loops/main.js

```diff
@@ -0,0 +1,69 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var number = [];
+  var currentNumber = (1);
+  while (currentNumber < 11) {
+    number.push(currentNumber);
+    currentNumber++;
+
+  }
+  return number;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = (0);
+  while (currentNumber < 20) {
+    currentNumber += 2;
+
+    evenNumbers.push(currentNumber);
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = (1);
+  var repeated = ' ';
+  while (count < times) {
+    count++;
+    repeated += word;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] * 2);
+  }
+  return doubled;
+}
+
+function getKeys(object) {
+  var keys = [];
+  for (var key in object) {
+    keys.push(`${key}`);
```

Comment:
you can just push the `key` variable here, it doesn't need to be converted to a string.

Reply:
ok thank you for letting me know